app/models/user.rb

```diff
@@ -81,6 +82,18 @@ def self.get_taskees_by_task_name(keyword, user_email = nil)
     taskees.where("email != ?", user_email)
   end
 
+  def novice?
```

Comment:
👍 On creating all these utility methods. Now it's easy to know if a user is in a specific plan or not.